ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -621,4 +624,12 @@ private void executeDDLCommand(DDLCommand ddlCommand) {
         );
     return avroCheckResult;
   }
+
+  private KsqlEntity handleRunScript(Map<String, Object> streamsProperties) {
```

Comment:
I don't see this being called anywhere. And it doesn't seem to actually be doing anything. Leftover?

Reply:
Left over, removed it.